site/_data/versions.yml

```diff
@@ -0,0 +1,4 @@
+vega: 5.27.0
```

Comment:
We generate this at build time so it doesn't need to be in the repo.

Reply:
Thanks, would it make sense for me to add this to the file in the `.gitignore`?

Reply:
Yes but make sure it doesn't cause issues with the page build.